File: LayoutTests/http/tests/misc/redirect-with-quotes.php
<?php
  // Test for various syntaxes of Refresh header, 
  // <http://bugzilla.opendarwin.org/show_bug.cgi?id=9854>.

  header('Content-type: text/html');
  header('Refresh: 0, URL 	= 	"resources/redirect-step2.php"');
?>

<body>
<script>
   if (window.testRunner) {
       testRunner.waitUntilDone();
       testRunner.dumpAsText();
   }
</script>
   
<p>FAILURE - should redirect (1)<p>
</body>

File: LayoutTests/http/tests/misc/redirect-with-quotes.py
#!/usr/bin/env python3

import sys

sys.stdout.write(
    'Refresh: 0, URL    =   "resources/redirect-step2.py"\r\n'
    'Content-Type: text/html\r\n\r\n\n'
    '<body>\n'
    '<script>\n'
    '   if (window.testRunner) {\n'
    '       testRunner.waitUntilDone();\n'
    '       testRunner.dumpAsText();\n'
    '   }\n'
    '</script>\n'
    '   \n'
    '<p>FAILURE - should redirect (1)<p>\n'
    '</body>\n'
)